pkg/txn/client/operator.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/matrixorigin/matrixone/pkg/pb/txn"
```

Comment:
You can not add tracing here.  You should add trace in the txn/rpc/sender and server.go